requirements_dev.txt

```diff
@@ -1,12 +1,12 @@
-pip==19.2.3
-bump2version==0.5.11
-wheel==0.33.6
-watchdog==0.9.0
-flake8==3.7.8
-tox==3.14.0
-coverage==4.5.4
-Sphinx==1.8.5
-twine==1.14.0
-Click==7.0
-pytest==4.6.5
-pytest-runner==5.1
\ No newline at end of file
+pip==22.0.4,
+bump2version==1.0.1,
+wheel==0.37.1,
+watchdog==2.1.6,
+flake8==4.0.1,
+tox==3.24.5,
+coverage==6.3.2,
+Sphinx==4.4.0,
+twine==3.8.0,
+Click==8.0.4,
```

Comment:
i might have missed this last week, how did you figure out which versions to change all these to?

Reply:
i went full send and updated all of them